charts/velocity-operator/templates/deployment.yaml

```diff
@@ -74,6 +74,8 @@ spec:
           - name: VELOCITY_API_URL
             value: {{ quote .Values.velocity.apiUrl }}
           {{- end }}
+          - name: REGISTRY_ENABLED
+            value: {{ quote .Values.registry.enabled }}
```

Comment:
Customers upgrading the helm chart will not have this value. You might want to add a | default "false" afterwards to cope with that